bot/utils/services.py

```diff
@@ -47,7 +47,13 @@ async def send_to_paste_service(contents: str, *, extension: str = "") -> Option
             continue
         elif "key" in response_json:
             log.info(f"Successfully uploaded contents to paste service behind key {response_json['key']}.")
-            return URLs.paste_service.format(key=response_json['key']) + extension
+
+            paste_link = URLs.paste_service.format(key=response_json['key']) + extension
+
+            if extension == '.py':
```

Comment:
Shouldn't extension not have a `.` here?
```suggestion
            if extension == 'py':
```

Reply:
A `.` is tacked onto the extension, if one is given beforehand.

https://github.com/python-discord/bot/blob/4be90b3c454138e3548c7394fcb2a1182b05b7d7/bot/utils/services.py#L22

Reply:
Aha, gotcha.